game/tests.py

```
@@ -1474,6 +1489,205 @@ def test_promotion_notation_invalid_piece_defaults_to_queen(self):
+
+    @override_settings(
+        DEBUG=True,
+        EMAIL_HOST_USER='',
+        EMAIL_HOST_PASSWORD='',
+    )
+    def test_responses_are_identical_for_existing_and_new_emails(self):
+        """Active-conflict and fresh registrations must produce the same status code."""
+        User.objects.create_user(
+            username='existingplayer',
+            email='existing@example.com',
+            password='StrongPass123!',
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "password"

(S106)